examples/integrations/sam/meta_sam_labelbox_video.ipynb

```diff
@@ -0,0 +1,1001 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #7.                                    queue_mode=lb.QueueMode.Batch)
</pre><p>We don't need to set the queue_mode. Should default to batch</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Labelbox/labelbox-python/pull/1123/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='25'/>